test/js/computed-style-property-map.js

```
@@ -8,20 +8,18 @@ suite('Computed StylePropertyMap', function() {
     document.body.removeChild(this.element);
   });
 
-  test('Computed StylePropertyMapReadOnly.get works for properties that can be NumberValues', function() {
+  test('get method returns a null object if the property is not supported', function() {
     var computedStyleMap = getComputedStyleMap(this.element);
-    var opacity = computedStyleMap.get('opacity');
-    assert.instanceOf(opacity, NumberValue);
-    assert.equal(opacity.value, 0.5);
-    assert.equal(opacity.cssString, '0.5');
+    var propertyStyleValue = computedStyleMap.get('lemon');
+
+    assert.equal(propertyStyleValue, null);
```

Let's throw an exception here instead of just returning null.